monitoring/boot_config_linux.go

```diff
@@ -110,31 +110,35 @@ func KernelVersionLessThan(version KernelVersion) KernelConstraintFunc {
 			(testVersion.Release == version.Release &&
```

Comment:
```go
func KernelVersionLessThan(version KernelVersion) KernelConstraintFunc {
	return func(testVersion KernelVersion) bool {
		release := testVersion.Release <= version.Release
		major := testVersion.Major <= version.Major
		minor := testVersion.Minor <= version.Minor
		patch := testVersion.Patch < version.Patch
		return release && major && minor && patch
	}
}
```
otherwise `3.1.10-1125` < `3.0.10-1126`

Reply:
Thanks for catching this. I've added a few extra test cases.